### PR TITLE
Update vim-01-beginner.tutor [RDY]

### PR DESCRIPTION
**Pull request by juliancoffee**
Command _:vim tutor_ I change to _:nvim tutor,_ because it is Neovim)

### CODE REVIEW

File: runtime/tutor/en/vim-01-beginner.tutor [hunk 1]
```diff
@@ -159,9 +159,9 @@ There is also some text missing here.
 
  2. At the shell prompt type this command:
 ~~~ sh
-     $ vim tutor
+     $ nvim tutor
 ~~~
-    'vim' is the command to start the Vim editor, 'tutor' is the name of
+    'nvim' is the command to start the Vim editor (it is NeoVim, but we call it as Vim sometimes), 'tutor' is the name of
```
Comment (justinmk) [inline runtime/tutor/en/vim-01-beginner.tutor]: "Neovim" not "NeoVim".

Comment (justinmk) [reply runtime/tutor/en/vim-01-beginner.tutor]: Actually just change this line to:

    nvim' is the command to start the Nvim editor, 'tutor' is the name of

**Comment by fmoralesc** [comment]
@juliancoffee Thanks for the changes. LGTM after @justinmk's suggestions.

File: runtime/tutor/en/vim-01-beginner.tutor (hunk 1 above)
Comment (juliancoffee) [reply runtime/tutor/en/vim-01-beginner.tutor]: Done

**Pull request merged**